contrib/offline/README.md

```diff
@@ -1,28 +1,30 @@
 # Offline deployment
 
-## manage-offline-container-images.sh
-
-Container image collecting script for offline deployment
+## tl;dr
```

Comment:
Does this mean `Quick start`?

Reply:
Steps in short description. Updated to "Steps".